### PR TITLE
Minor Contribution on Finding Grammatical Error.

### PR DESCRIPTION
**Pull request by melvinljy96**
As a beginner, I try to make a contribution from minor part like finding grammatical errors and adding descriptions as my first try.

### CODE REVIEW

**Comment by melvinljy96** [comment]
may i know my contribution merged?
Your kind consideration is much appreciated.
Thank you for your attention.

**Comment by nicolasnoble** [comment]
I'll let the people responsible for the documentation to review and decide. They are on California time, so it's the middle of the night right now anyway :-)

**Comment by melvinljy96** [comment]
Okay, sir. Thank you for your reply.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 gRPC - An RPC library and framework
 ===================================
 
-gRPC is a modern, open source, high-performance remote procedure call (RPC) framework that can run anywhere. It enables client and server applications to communicate transparently, and makes it easier to build connected systems.
+gRPC is a modern, open source, high-performance remote procedure call (RPC) framework that can be used and run anywhere and anytime. It not only enables client and server applications to communicate transparently, but also makes it easier to build connected systems.
```
Comment (srini100) [inline README.md]: Replace "be used and run..." with "be used in variety of [languages and platforms](https://grpc.io/about/#osp)."

File: README.md [hunk 2]
```diff
@@ -43,21 +44,20 @@ Precompiled bleeding-edge package builds of gRPC `master` branch's `HEAD` are up
 
 Contributions are welcome!
 
-Please read [How to contribute](CONTRIBUTING.md) which will guide you through the entire workflow of how to build the source code, how to run the tests and how to contribute your changes to
-the gRPC codebase.
+Please read [How to contribute](CONTRIBUTING.md) which will guide you through the entire workflow of how to build the source code, how to run the tests and how to contribute your changes tothe gRPC codebase.
```
Comment (srini100) [inline README.md]: Fix "tothe"

File: README.md [hunk 3]
```diff
@@ -43,21 +44,20 @@ Precompiled bleeding-edge package builds of gRPC `master` branch's `HEAD` are up
...
 The document also contains info on how the contributing process works and contains best practices for creating contributions.
 
 # Troubleshooting
 
-Sometimes things go wrong. Please check out the [Troubleshooting guide](TROUBLESHOOTING.md) if you are experiencing issues with gRPC.
+If you find out there are errors or issues, feel free to check out the [Troubleshooting guide](TROUBLESHOOTING.md) if you are experiencing issues with gRPC.
```
Comment (srini100) [inline README.md]: Change to "If you are experiencing issues with gRPC, please checkout the [Troubleshooting guide](TROUBLESHOOTING.md)."

File: README.md [hunk 4]
```diff
@@ -43,21 +44,20 @@ Precompiled bleeding-edge package builds of gRPC `master` branch's `HEAD` are up
...
 
 # Performance 
 
 See [Performance dashboard](http://performance-dot-grpc-testing.appspot.com/explore?dashboard=5636470266134528) for the performance numbers for the latest released version.
 
 # Concepts
 
-See [gRPC Concepts](CONCEPTS.md)
+Check out [gRPC Concepts](CONCEPTS.md) to understand more about the concept of gRPC.
```
Comment (srini100) [inline README.md]: concepts

**Comment by vjpai** [comment]
Closing as obsolete since no response in >1 month.

**Pull request closed without merging**